### ISSUE TITLE
Wallrunning poses are shifted by 16px

### ISSUE THREAD

**Issue by TheLX5** [TheLX5/Powerups#3]
I don't even know what caused it, it started happening between 3.0.4 and 3.1.0 and GitHub didn't notice any relevant changes in the relevant files that handles Mario's poses and interaction.

**Comment by SubconsciousEye** [comment]
I decided to test this by editing some graphics a tiny bit (adding a strip of pixels at the left/rightmost edge of a 32x32 pose) and forcing $7E13E0 to certain poses. I also tested these poses during a wallrun and during a standstill (in other words, not moving at all on flat ground).
It appears that the actual position of the pose graphics aren't being updated when necessary, so that might be something to look in to. It's possible that there may be missing code, or code not jumping to the right place, or code that is doing stuff that it shouldn't.
I haven't tested anything outside of these conditions, so it might be a good idea to test those (like certain player states that are possible alongside wallrunning).

![wallruntest_00000](https://user-images.githubusercontent.com/5531548/39482400-88eddb88-4d34-11e8-95ad-35443f8ced1c.png)
![wallruntest_00001](https://user-images.githubusercontent.com/5531548/39482401-88ff237a-4d34-11e8-9443-5f2dee0cc3b2.png)
![wallruntest_00002](https://user-images.githubusercontent.com/5531548/39482402-891c4626-4d34-11e8-985e-8c20c3b58dce.png)
![wallruntest_00003](https://user-images.githubusercontent.com/5531548/39482403-892dbb40-4d34-11e8-8617-fdf34d0300ea.png)
